package/environments/base.js

```diff
@@ -111,7 +112,12 @@ module.exports = class Base {
     this.resolvedModules = getModulePaths()
   }
 
-  toWebpackConfig() {
+  toWebpackConfig(path) {
+    this.loaders = getLoaderList()
```

Comment:
Had to reset these properties here. Something to do with the parallelization was stripping them of being ConfigObjects